src/compiler/commandLineParser.ts

```diff
@@ -505,6 +505,12 @@ export const commonOptionsWithBuild: CommandLineOption[] = [
         description: Diagnostics.Set_the_language_of_the_messaging_from_TypeScript_This_does_not_affect_emit,
         defaultValueDescription: Diagnostics.Platform_specific
     },
+    {
+        name: "skipJSDocParsing",
+        type: "boolean",
+        defaultValueDescription: false,
+        hiddenInAllViews: true,

```

Comment:
This needs to have `affectsSourceFile` true so that it is used in caching sourceFiles correctly.

Reply:
I can add that flag, though I don't think it actually matters because this flag is never legal to pass outside of `tsc`, i.e. tsserver or the libraries should never set this.